src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { DELAY_TO_CHANGE_DATE } from '../../utils/vars';
+
+type TClockProps = {
+  clockName: string;
+};
+
+type TState = {
+  today: Date;
+};
+
+export class Clock extends React.PureComponent<TClockProps, TState> {
+  state = {
+    today: new Date(),
+  };
+
+  todayTimerId = 0;
+
+  componentDidMount(): void {
+    this.startTimerDate();
+  }
+
+  componentDidUpdate(
+    prevProps: Readonly<TClockProps>,
+    prevState: Readonly<TState>,
+  ): void {
+    if (prevProps.clockName !== this.props.clockName) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevProps.clockName} to ${this.props.clockName}`);
+    }
+
+    if (prevState.today !== this.state.today) {
+      // eslint-disable-next-line no-console
+      console.info(this.state.today.toUTCString().slice(-12, -4));
```

Comment:
You can move this expression to a separate function like getTodayInUTC and you will use it also on 60th line

Reply:
Good point